en/simulation/hitl.md

```diff
@@ -26,7 +28,35 @@ These parameter are optional and allow to run HITL without a radio remote contro
 
 ## Connect to the Simulator
 
-Currently either jMAVSim or X-Plane and QGroundControl are supported. Follow either setup step below.
+Currently Gazebo, jMAVSim, or X-Plane and QGroundControl are supported. Follow either setup step below.
+
+### Using Gazebo
+* Update the environment variables: 
+``` 
+cd <Firmware_clone>
+make posix_sitl_default gazebo
+source Tools/setup_gazebo.bash $(pwd) $(pwd)/build/posix_sitl_default
+```
+
+* Open the vehicle model's sdf file (e.g. Tools/sitl_gazebo/models/iris/iris.sdf). 
```

Comment:
Iris is a [Quadrotor Wide](https://docs.px4.io/en/airframes/airframe_reference.html#quadrotor-wide) - does it matter that there is a mismatch between this model and the options you can select in QGC (Quadrotor + and x)?

Reply:
It does not appear to make a difference.